pkg/kubectl/cmd/clusterinfo_dump_test.go

```diff
@@ -33,7 +34,7 @@ func TestSetupOutputWriterNoOp(t *testing.T) {
 		f := cmdtesting.NewTestFactory()
 		defer f.Cleanup()
 
-		cmd := NewCmdClusterInfoDump(f, os.Stdout)
+		cmd := NewCmdClusterInfoDump(f, genericclioptions.IOStreams{Out: os.Stdout, ErrOut: os.Stderr})
```

Comment:
There's `genericclioptions.NewTestIOStreamsDiscard` that you can use here, since you don't care about the outputs.